app/client/src/features/events/Questions/QuestionList/QuestionList.tsx

```diff
@@ -185,9 +186,10 @@ export function QuestionList({ className, style, fragmentRef }: Props) {
                                                             : { width: '100%', maxWidth: '10rem' }
                                                     }
                                                     className={classes.questionActions}
-                                                    like={!isModerator && Boolean(user)}
-                                                    quote={!isModerator && Boolean(user)}
-                                                    queue={isModerator && Boolean(user)}
+                                                    likeEnabled={!isModerator && Boolean(user)}
```

Comment:
Casting too boolean here is redundant.  Although it doesn't look like you changed this line of code.  Maybe this was formatting?

Reply:
It's only redundant because normally it would be 1 question list, but it is two so the moderator can have infinite scroll like Kevin wanted. Once we updated to the react virtualized it will be necessary again. That's why I left it in.

Reply:
Err, I'm not sure that's what I meant.  I meant that casting a `user` to a boolean is redundant.  So, the statements could just be `!isModerator && user` rather than `Boolean(user)` since, because of type coercion, the `user` variable will already be casted to a Boolean.

Reply:
Looking at the commit history, it looks like I added `Boolean(user)`, oops. Feel free to ignore this comment.